lib/messenger_old.rb

```diff
@@ -1,29 +0,0 @@
-# This class tries to mimic ActiveMailer interface, but sends internal messages
-# insteads of emails.
-#
-# TODO: make this as close to ActiveMailer as possible. Support templates,
-# helpers, and so on.
-class Messenger
```

Comment:
I guess you had your reasons to remove this. I trust you :+1:

Reply:
I removed because it looked like no place was using it. 
@hallelujah @guicassolato any idea if this file is being used somewhere?